### DISCUSSION TITLE
QSelect Click Propagation

### DISCUSSION THREAD

**Discussion by j-wenning** [quasarframework/quasar#9887] [General - Components / Directives / etc] [answered]
Click propagation is not captured/prevented by `q-select` when behavior is set to `dialog`.  `@click` is never emitted by q-select.  Is this intended?  If so: why?

**Example**
https://codepen.io/j-wenning/pen/qBmELGY

<img width="1440" alt="Screen Shot 2021-06-30 at 2 29 13 PM" src="https://user-images.githubusercontent.com/58379242/124034738-7415f680-d9b0-11eb-8964-f430d2b6a92e.png">

**To Reproduce**
1. Open the inspector
2. Click the left select (menu behavior)
3. Observe (lack of) console output
4. Click the right select (dialog behavior) 
5. Observe console output

**Comment by pdanpdan** [comment, marked as answer]
I've tracked it to the source, thank you for reporting.

https://github.com/quasarframework/quasar/pull/9895
